CONTRIBUTING.md

```diff
@@ -0,0 +1,167 @@
+# How to contribute
+
+We are very happy to have you contributing to our common effort!
```

Comment:
could start by outlining different ways to contribute, e.g. user stories, pull requests, feature requests, fixing bugs, writing tests ... starting with ways that non-technical people can contribute

Reply:
Added